public/app/features/dashboard-scene/settings/DashboardLinksEditView.tsx

```diff
@@ -0,0 +1,64 @@
+import React from 'react';
+
+import { PageLayoutType } from '@grafana/data';
+import { locationService } from '@grafana/runtime';
+import { SceneComponentProps, SceneObjectBase } from '@grafana/scenes';
+import { Link } from '@grafana/ui';
+import { Page } from 'app/core/components/Page/Page';
+
+import { NavToolbarActions } from '../scene/NavToolbarActions';
+import { dashboardSceneGraph } from '../utils/dashboardSceneGraph';
+
+import { EditListViewSceneUrlSync } from './EditListViewSceneUrlSync';
+import { DashboardEditView, DashboardEditListViewState, useDashboardEditPageNav } from './utils';
+
+export interface DashboardLinksEditViewState extends DashboardEditListViewState {}
+
+export class DashboardLinksEditView extends SceneObjectBase<DashboardLinksEditViewState> implements DashboardEditView {
+  static Component = DashboardLinksEditViewRenderer;
+
+  protected _urlSync = new EditListViewSceneUrlSync(this);
+
+  public getUrlKey(): string {
+    return 'links';
+  }
+
+  getDashboardLinks() {
+    const dashboard = this.state.dashboardRef.resolve();
+    return dashboardSceneGraph.getDashboardLinks(dashboard);
```

Comment:
What do you think of moving the links to the root DashboardScene? 

And have DashboardLinksControls just do:

```ts
  const { links, uid } = getDashboardFor(model).useState();
```

Feels a bit awkward to have the links nested inside this view component and it has dashboardUID state. I think state wise the links are it's part of the top-level dashboard. would also make scene to save model easier.

Reply:
think this is a good idea actually.